### ISSUE TITLE
Cannot read property 'split' of undefined

### ISSUE THREAD

**Issue by zyadamec** [ReadableLabs/readable#1]
**Describe the bug**
While commenting Python I get the error "Cannot read property 'split' of undefined"

**To Reproduce**
1. Highlight code below(Not sure why the code block isn't keeping the formatting.  Code format can bee seen here) https://imgur.com/a/tQGQOGv

`
def aws_heading_style():
    heading = NamedStyle(name="heading")
    #heading.font = Font(name='Segoe UI Light', size=11, bold=True)
    heading.number_format = '$#,##0.00'
    heading.fill = PatternFill(
        fill_type='solid',
        start_color='DDEBF7',
        end_color='DDEBF7')
    return heading
`

3. Click Ctrl + '
4. Error shows

**Expected behavior**
Comment generated

**Screenshots**
https://imgur.com/a/tQGQOGv

**Desktop (please complete the following information):**
 - OS: Windows 10
 - VSCode Version:
 https://imgur.com/a/tQGQOGv
 - Readable Version: 1.5.5

**Comment by tippf3hlr** [comment]
Had the same issue, but it only appears when the code block is highlighted. If I press ctrl + ' while having the cursor in the function works fine.

**Comment by iamchadarmstrong-pki** [comment]
I can confirm the same behaviour for Typescript

**Comment by MattRiddell** [comment]
Yeah I'm getting that today as well in Typescript - nothing hilighted.

**Comment by dsetzer** [comment]
I've been getting that started when my trial ended if I try to generate a comment for anything.
